web-share/src/index.html

```diff
@@ -10,6 +10,9 @@
     <link rel="stylesheet" href="css/svelte.css">
     <script defer src="js/app.js"></script>
     <link rel="manifest" href="manifest.json">
+    <!-- This Origin Trial token for Content Indexing will only work when deployed to https://scrapbook-pwa.web.app/ -->
```

Comment:
I've tried https://scrapbook-pwa.web.app and `registration.index` is `undefined`. Is this intended?

Reply:
Could it simply be that this latest version is not deployed yet?

Reply:
You need Chrome 80+ on Android.

I just confirmed that with the current Canary on Android (M81) and the origin trial token is working for https://scrapbook-pwa.web.app/